src/Functions/largestConnCompSliceWise.m

```diff
@@ -1,6 +1,20 @@
-% Andreas Husch
-% 2018
 function binaryMask = largestConnCompSliceWise(img3d)
+%
+% USAGE:
+%
+%    binaryMask = largestConnCompSliceWise(img3d)
+%
+% INPUT: 
+%    img3d:     
```

Comment:
@adhusch Could you please provide a description for the input and output?

Reply:
Please see my comment from 2 days ago